app/src/componentLibrary/radioGroup/radioButton/radioButton.jsx

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright 2022 EPAM Systems
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import styles from './radioButton.scss';
+
+const cx = classNames.bind(styles);
+
+export const RadioButton = ({ option, value, onChange, onFocus, onBlur, className, variant }) => {
+  const isChecked = option.value === value;
+
+  return (
+    // eslint-disable-next-line
+    <label
+      className={cx(variant, className, 'radio-button', {
+        disabled: option.disabled,
+      })}
+      onFocus={onFocus}
+      onBlur={onBlur}
+      tabIndex="1"
```

Comment:
Please make sure that's work as expected. Looks like you override default behaviour and RadioButton will be selected at first even it is last input on the page
https://webaim.org/techniques/keyboard/tabindex